co2eq/energyprices.yml

```diff
@@ -0,0 +1,222 @@
+_children:
```

Comment:
remove all the children keys, the are not needed here

Reply:
ok - would love an explanation of what purpose _children has :D